app/src/test/java/com/codesoom/assignment/controllers/ProductControllerTest.java

```diff
@@ -0,0 +1,199 @@
+package com.codesoom.assignment.controllers;
+
+import com.codesoom.assignment.ProductNotFoundException;
+import com.codesoom.assignment.application.ProductService;
+import com.codesoom.assignment.domain.Product;
+import com.codesoom.assignment.domain.ProductRepository;
+import com.codesoom.assignment.infra.InMemoryRepository;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+class ProductControllerTest {
+
+    private static final String PRODUCT_MAKER = "test1";
+    private static final String PRODUCT_NAME = "딸랑이";
+    private static final String PRODUCT_MAKER2= "test2";
+    private static final String UPDATE_POSTFIX = "!!!";
+    private static final Long INVALID_ID = 0L;
+    private static final Long VALID_ID = 1L;
+
+    private ProductController controller;
+    private ProductService productService;
+
+    @BeforeEach
+    void setUp() {
+        ProductRepository productRepository = new InMemoryRepository();
+
+        productService = new ProductService(productRepository);
+        controller = new ProductController(productService);
+
+        Product product = new Product();
+        product.setMaker(PRODUCT_MAKER);
+
+        controller.create(product);
+    }
+
+    @Nested
+    @DisplayName("list 메서드는")
+    class Describe_list{
+        @Nested
+        @DisplayName("등록된 Product가 있다면")
+        class Contest_have_product{
+
+            @Test
+            @DisplayName("등록된 모든 Product를 리턴한다")
+            void It_return_list(){
+                List<Product> products = controller.list();
+                assertThat(products).hasSize(1);
+                assertThat(products.get(0).getMaker()).isEqualTo(PRODUCT_MAKER);
+
```

Comment:
```suggestion
                List<Product> products = controller.list();
                
                assertThat(products).hasSize(1);
                assertThat(products.get(0).getMaker()).isEqualTo(PRODUCT_MAKER);
```

적절히 빈줄로 구분해주면 좋을 것 같아요